docs/pages/deploy-a-cluster/teleport-enterprise/getting-started.mdx

```diff
@@ -1,465 +1,489 @@
-  - You can stop the Teleport container using `docker-compose -f teleport-ent-quickstart.yml down`
+## Next steps
 
-### Quickstart using docker run
+### Deploy on Kubernetes
 
-If you'd prefer to complete these steps manually, here's some sample `docker run` commands:
+This guide shows you how to install Teleport Enterprise on a virtual machine. If
+you are using a Kubernetes-based environment, see our [Getting Started
+Guide](../getting-started/kubernetes-cluster.mdx) for how to deploy Teleport on
+Kubernetes.
 
-```code
-# create local config and data directories for teleport, which will be mounted into the container
-$ mkdir -p ~/teleport/config ~/teleport/data
-
-# download your license file from the Gravitational dashboard and put it in the correct directory
-# the file needs to be named license.pem
-$ cp ~/downloads/license.pem ~/teleport/data/license.pem
-
-# generate a sample teleport config and write it to the local config directory
-# this container will write the config and immediately exit - this is expected
-$ docker run --hostname localhost --rm \
-  --entrypoint=/bin/sh \
-  -v ~/teleport/config:/etc/teleport \
-  (=teleport.latest_ent_docker_image=) -c "teleport configure > /etc/teleport/teleport.yaml"
-
-# change the path to the license file in the sample config
-$ sed -i 's_/path/to/license-if-using-teleport-enterprise.pem_/var/lib/teleport/license.pem_g' ~/teleport/config/teleport.yaml
-
-# start teleport with mounted license, config and data directories, plus all ports
-$ docker run --hostname localhost --name teleport \
-  -v ~/teleport/config:/etc/teleport \
-  -v ~/teleport/data:/var/lib/teleport \
-  -p 3023:3023 -p 3025:3025 -p 3080:3080 \
-  (=teleport.latest_ent_docker_image=)
-```
+### Configure RBAC
 
-### Creating a Teleport user when using Docker quickstart
+The preset `access` role we assigned to a user in this guide is probably too
+permissive for your environment. Read our guide to [configuring Teleport
+roles](../../access-controls/guides/role-templates.mdx) to set up more granular
+access controls.
 
-To create a user inside your Teleport Enterprise container, use `docker exec`.
+### Configure SSO
 
-This example command will create a Teleport user called `testuser` which has the
-`access` role. Feel free to change these to suit your needs.
+In this guide, we created a **local user** stored on the Teleport Auth Service.
+For on– and offboarding users at scale, you should use one of Teleport's Single
+Sign-On integrations.
 
-```code
-$ docker exec teleport tctl users add --roles=access --logins=root,ec2-user,ubuntu testuser
-```
+Take a look at our [Single Sign-On](sso.mdx) guide to learn the basics of
+integrating Teleport with SSO providers.
 
-When you run this command, Teleport will output a URL which you must open to complete the user signup process:
+You can configure any SAML– or OIDC-compliant identity provider to enable SSO
```

Comment:
I feel it may make more sense to use `SAML` instead of `SAML-`

Reply:
For this one, I think it's a bit clearer to show that "SAML" is also part of the same compound modifier as "OIDC". I can use a hyphen instead of a dash here, though.